client/packages/coldchain/src/Monitoring/ListView/TemperatureChart/TemperatureChart.tsx

```diff
@@ -177,6 +177,7 @@ const Chart = ({
                       borderStyle: 'solid',
                       borderColor: theme.palette.gray.light,
                       padding: 3,
+                      textAlign: 'left',
```

Comment:
Offroad: the legend was showing with names centred:

<img width="273" alt="Screenshot 2023-12-06 at 10 24 39 AM" src="https://github.com/msupply-foundation/open-msupply/assets/9192912/b581b61a-7eac-40fb-87fc-e2be71aa98fe">